### ISSUE TITLE
Why are we CSS Grid first?

### ISSUE THREAD

**Issue by dr3** [bbc/simorgh#2555]
**Question**
We are CSS Grid first. Our primary implementation is CSS Grid and our non-grid implementations are an after thought, should they be?

As simorgh pivots to primarily WS is there a need to use both Grid and non-Grid implementations? What if we just used Grid, or even flex, for 'nice to have' style improvements with the primary layout being not grid?

With WS as the focus the need for full non grid support seems like a strong requirement, would it save dev effort just dropping it and focusing on non-grid only?

What are our support numbers for Grid/non grid? Are they likely to change in the next year to a point where we drop non-grid support? 

What about web core? If they reuse psammead components, will they need/want grid?

**Disclaimer:**
Just a thought, not suggesting either way but this question has come up a few times recently, and if we decide we can drop grid, it will save us a lot of dev effort

**Testing:**
Whatever solution we go for, this should have a developer accessibility swarm carried out on it to ensure the CSS does not affect assistive technology

**Comment by jamesbhobbs** [comment]
The ideal fallback would be table based layout, which is universally supported and would fit very nicely with grid doing the enrichment on top. Such an approach is common on things like emails. However, as a young dev team I think this could be challenging in terms of table layout skill and accepting the idea as anything other than horrifying.

I use my words relatively carefully, the grid would still be primary, but you'd write it in an enriching way. If it's of interest I can dig out some examples but honestly I'd rather see ideas of making our current solution better than reinventing the wheel.

**Comment by pjlee11** [comment]
I think we should be looking at the stats in this case. WS audiences are/were (need to check again) predominately mobile so we should follow a mobile first implementation. We previously implemented CSS grid on articles because at wide viewports we had complicated requirements with full-bleed images and the onward journey to the right of the page. This was not a mobile first implementation. 

As both the OJ and full-bleed images are not current requirements I don't see the need for using CSS grid if we have an implementation that works without grid. Otherwise we are supporting a fallback that a significant amount of our users see, and a second implementation that we will see and dev against daily. 

TL;DR: If we can achieve the complex layout we required without CSS grid then I think we don't need grid.

**Comment by pjlee11** [comment]
Forking @dr3's [codepen](https://codepen.io/drewmcmillan/pen/MMMZwB?editors=0010) and removing the use of CSS grid (and adding some padding of visual ease) shows that we can achieve nested layout logic in terms of the amount of columns an element should span without using CSS grid. [Forked codepen](https://codepen.io/phil-lee/pen/mZNrRE?editors=0010)

**Comment by jamesbhobbs** [comment]
This is now covered in other issues, the shared components pod will rewrite all of our current layout methods then move them to psammead repo
